pkg/tfgen/docs.go

```diff
@@ -822,82 +821,166 @@ func (p *tfMarkdownParser) parseSchemaWithNestedSections(subsection []string) {
 
 // parseArgFromMarkdownLine takes a line of Markdown and attempts to parse it for a Terraform argument and its
 // description
-func parseArgFromMarkdownLine(line string) (string, string, bool) {
+func parseArgFromMarkdownLine(line string) (string, string, bool, bool) {
```

Comment:
I have no idea what this returns. Can we add a type here?
```suggestion
func parseArgFromMarkdownLine(line string) (struct{ bullet: string, text: string, indentedBullet: bool}, bool) {
```

I'm not sure the labels that I suggested are correct (which is kinda the point), but I think we need clarity here.